### PR TITLE
Update Third Party License text

### PR DESCRIPTION
**Pull request by m-aXimilian**
## Description

PR #22 added a dependency to rapdijson which has to be declared in the third party license text.

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
no functional changes -> no tests required

## Checklist:

- [x] I followed the Contributing Guidelines.
- [x] I did a self-review.
- [ ] I commented my code, particularly in hard-to-understand areas.
- [ ] I updated the documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.

### CODE REVIEW

File: THIRD_PARTY_LICENSES.txt [hunk 1]
```diff
@@ -1220,3 +1220,41 @@ HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
 WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 OTHER DEALINGS IN THE SOFTWARE
+
+
+-------------------------------------------------------------------------------
+|  pugixml 815e6e7 (used under MIT License) http://pugixml.org/
```
Comment (DaveyJonesBitPail) [inline THIRD_PARTY_LICENSES.txt:1226]: I am assuming this was intended to be rapidjson rather than pugixml which we already have an entry for

Comment (m-aXimilian) [reply THIRD_PARTY_LICENSES.txt]: Oh my...of course thank you!

File: THIRD_PARTY_LICENSES.txt [hunk 2]
```diff
@@ -1220,3 +1220,41 @@ HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
 WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
 FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 OTHER DEALINGS IN THE SOFTWARE
+
+
+-------------------------------------------------------------------------------
+|  RapidJSON 815e6e7 (used under MIT License) http://pugixml.org/
```
Comment (DaveyJonesBitPail) [inline THIRD_PARTY_LICENSES.txt:1226]: We may want to just stick to the recommended license text on the readme:
https://github.com/Tencent/rapidjson?tab=readme-ov-file#copyright-and-licensing

Comment (DaveyJonesBitPail) [inline THIRD_PARTY_LICENSES.txt:1226]: URL link would also need to be updated

Comment (m-aXimilian) [reply THIRD_PARTY_LICENSES.txt]: embarrassing 🙈
sorry and thanks!!

Comment (m-aXimilian) [reply THIRD_PARTY_LICENSES.txt]: Done [here](https://github.com/ZEISS/czicheck/pull/24/commits/3a118ad9e3efd930666d7fe2c889a39bfcc0d276)

**Review by DaveyJonesBitPail — APPROVED** [review]
lgtm!

**Pull request merged**